export/eventrelay-api/.env.example

```
@@ -0,0 +1,20 @@
+# EventRelay API configuration. All vars use the EVENTRELAY_ prefix.
+# Copy to .env for local use (.env is gitignored). Unset values fall back to
+# safe local defaults (in-memory store, no live model calls).
+
+# ── App ──
+EVENTRELAY_APP_NAME=EventRelay API
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)